dm/dm-ansible/roles/dm-worker/templates/run_dm-worker.sh.j2

```diff
@@ -5,12 +5,26 @@ ulimit -n 1000000
 DEPLOY_DIR={{ deploy_dir }}
 cd "${DEPLOY_DIR}" || exit 1
 
+{% set my_ip = hostvars[inventory_hostname].ansible_host | default(hostvars[inventory_hostname].inventory_hostname) -%}
+{% set my_port = hostvars[inventory_hostname].dm_worker_port -%}
+{% set my_separator = "_" %}
+{% set my_hostname = hostvars[inventory_hostname]['ansible_hostname'] | default(hostvars[inventory_hostname].inventory_hostname) -%}
+{% set my_peer_id = my_separator ~ my_hostname ~ my_separator ~ my_port %}
+
+{% set all_dm_worker = [] -%}
```

Comment:
Is naming it as `all_dm_master` more accurate?

Reply:
`all_dm_master` is in `dm/dm-ansible/roles/dm-worker/templates/run_dm-master.sh.j2`

Reply:
these are `all_dm_master` which `dm_worker` is joining.

Reply:
Make sense, agree